src/elements/Element.js

```diff
@@ -541,9 +540,9 @@ export default class Element {
      *
      * @returns {Element}
      */
-    remove() {
+    remove(): ?Element {
         if (!this.parentElement) {
-            return this;
+            return;
```

Comment:
Ohsies, that is some news, i guess DOM is still incoherent even with new methods :/